src/app/templates/_.babelrc

```diff
@@ -0,0 +1,4 @@
+{
+  "presets": ["es2015-loose"],
```

Comment:
we probably should default to non-loose and maybe have an option for loose mode? Plugins that don't care or need IE8 don't want loose mode. Also, for IE8 support, we want the other two or three transforms from videojs.